source/frontend/src/components/contact/ContactManagerView/ContactManagerView.test.tsx

```diff
@@ -111,7 +110,7 @@ describe('ContactManagerView', () => {
   });
 
   it('searches by city/municipality', async () => {
-    setupMockSearch([{ ...defaultSearchResult, municipalityName: 'victoria' }]);
+    setupMockSearch([{ ...defaultPersonSearchResult, municipalityName: 'victoria' }]);
```

Comment:
we need to move away from static moc objects given some other section of the code could change it. instead, use a factory approach to have a method construct new objects on each call

Reply:
true, this isn't really part of this PR (pre-existing mock objects). This was just a re-name. I don't mind correcting this however.